src/component/AutoComplete.tsx

```
@@ -0,0 +1,133 @@
+      />
+
+      {isOpen && (filteredItems.length > 0 || showAddNewItem) && (
+        <div className="absolute z-50 mt-2 w-full rounded-lg border border-border bg-white shadow-lg">
+          {filteredItems.length > 0 && (
+            <div className="max-h-[300px] overflow-y-auto p-1">
+              {filteredItems.map((item) => (
+                <button
+                  key={item.id}
+                  onClick={() => handleSelect(item.name)}
+                  className="w-full text-left px-3 py-2 text-base hover:bg-accent rounded-md transition-colors"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)